### PR TITLE
Player-Link-Implementation

### PR DESCRIPTION
**Pull request by Samir2003**
Finished player class (FOR NOW <- We're not changing it now). Pretty scuffed but gg.

### CODE REVIEW

**Review by JacobIm28 — APPROVED** [review]
Looks good

**Review by JacobIm28 — APPROVED** [review]

**Review by JacobIm28 — CHANGES_REQUESTED** [review]
Shouldn't default value for the entries in abiltiiesrem be true.

File: src/model/player.cc [hunk 1]
```diff
@@ -5,13 +5,128 @@
...
+    if(links.at(i)->getLabel() == link){
+      return links.at(i)->getStrength(); 
+    }
+  }
+}
+
+Position Player::move(char link, Direction dir){
+  Position pos;
+  std::vector<Position> positions;
+  Position temppos;
```
Comment (JacobIm28) [inline src/model/player.cc:51]: U and ur stupid all lowercase naming convention

**Review by JacobIm28 — CHANGES_REQUESTED** [review]
Run make clean but everything else is good

**Review by JacobIm28 — APPROVED** [review]
look good

**Review by gnishiwaki — APPROVED** [review]
Looks good to me

**Pull request merged**